docs/README.rst

```diff
@@ -33,383 +33,21 @@ It contains the following types:
 * Link
 * News Item
 
-The main difference from a users perspective is that these types are editable and extendable through-the-web. This means you can add or remove fields and behaviors using the control-panel "Dexterity Content Types" (``/@@dexterity-types``).
-
-**Warning: Using plone.app.contenttypes on a site with existing Archetypes-based content requires migrating the sites content. Please see the chapter "Migration".**
+These types are editable and extendable through-the-web and you can add or remove fields and behaviors using the control-panel "Dexterity Content Types" (``/@@dexterity-types``).
 
 
 Compatibility
 -------------
 
-The versions 1.2.x (build from the master-branch) are used in Plone 5.
-
-Version 1.1b5 and later are tested with Plone 4.3.x. The versions build from the branch 1.1.x will stay compatible with Plone 4.3.x.
+Version 3 is build for Plone 6 only.
```

Comment:
Typo `build` -> `built`. 

But (just a suggestion) I would even remove the compatibility section given that this package is in Plone core for all the currently supported versions.

Reply:
removal makes sense to me